Explorer/Assets/DCL/Profiles/Self/ISelfProfile.cs

```diff
@@ -8,7 +8,7 @@ public interface ISelfProfile
     {
         UniTask<Profile?> ProfileAsync(CancellationToken ct);
 
-        UniTask<Profile?> PublishAsync(CancellationToken ct);
+        UniTask<Profile?> PublishAsync(CancellationToken ct, bool onlyBasicInfo = false);
```

Comment:
It is not clear in the contract what `onlyBasicInfo` means and also makes the implementation more complicated for a custom use case, violating single responsibility principle. I think its better to create an overload like: `PublishWithoutModifications(CancellationToken ct)`